bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx

```diff
@@ -81,7 +81,11 @@ class NavBar extends PureComponent {
       mountModal,
       presentationTitle,
       amIModerator,
+      name,
+      breakname,
```

Comment:
Naming should be better. Should be somethng like breakoutRoomName, instead of breakname